flow/metrics.yaml

```diff
@@ -19,6 +19,13 @@ components:
           type: boolean
           default: false
           x-field-uid: 2
+        rx_tx_ratio:
+          description: |-
+            Tx Rx multiplication factor, required for loss measurement. Should be a positive, non-zero value.
```

Comment:
It might be more intuitive to 
a) Call it tx_rx_replication_factor
b) Have a set of enums : 
    one
    number_of_rx_ports
    custom 
         Here , user can give a float if needed 
   The default being 'one'
   No of rx_ports means for replicating use-case of multicast , normally replication factor can be auto calculated and be same as number of rx ports.

Reply:
Update the property name and made it into an enum.